examples/monitoring/README.md

```diff
@@ -4,3 +4,7 @@ Steps:
 1. Run `python2 query.py`
 2. Wait for the query.py start outputing predictions.
 3. Run `python init_grafana.py` and follow the instruction.
+
+For Kubernetes:
+
+- You can find the metric address via `KubernetesContainerManager::get_metric_addr` function. 
```

Comment:
nit: change to "\`KubernetesContainerManager.get_metric_addr\`"

Reply:
Fixed